Tests/ML/datasets/test_dataset.py

```diff
@@ -419,6 +421,74 @@ def test_get_all_metadata(default_config: ModelConfigBase) -> None:
     assert PatientMetadata.from_dataframe(df, '2') == PatientMetadata(patient_id='2', institution="2")
 
 
+def test_converts_channels_to_file_paths(default_config: ModelConfigBase) -> None:
+    """
+    Test unit for missing channels and missing files.
+    """
+    # Sets DummyModel config and container
+    container = InnerEyeContainer(default_config)
+
+    # 1 Should not return any errors given that no channels or files are missing
+    container.setup()
+
+    # 2 Creates InnerEyeContainer object with: 1) missing channels: "channel1", "channel2", 'mask'
+    # for patients 1 and 7; 2) Wrong file name for channel "mask", patient 2. Error report is generated with the
+    # corresponding information.
+    data_frame = [
+        # ["1",  "train_and_test_data/id1_channel1.nii.gz", "channel1",  "1"],  
+        # ["1",  "train_and_test_data/id1_channel1.nii.gz", "channel2",  "1"],
+        # ["1",  "train_and_test_data/id1_mask.nii.gz",     "mask",      "1"],
```

Comment:
can you add a comment saying why this is commented out?

Reply:
Added comment explaining why is commented out.